packages/offix-scheduler/.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
```

Comment:
Let's avoid adding changes not relevant to PR

Reply:
Okay